### DISCUSSION TITLE
Slim Application Error A website error has occurred. Sorry for the temporary inconvenience.

### DISCUSSION THREAD

**Discussion by sahanasaakshi** [nextcloud/all-in-one#4566] [Questions] [answered]
I'm not able to install Nextcloud AIO getting the below error. Please help me on this.

NOTICE: PHP message: Slim Application Error
Type: Exception
Code: 0
Message: Could not create container nextcloud-aio-nextcloud: Client error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-nextcloud` resulted in a `400 Bad Request` response:
{&quot;message&quot;:&quot;invalid mount config for type \&quot;bind\&quot;: bind source path does not exist: /data/mnt&quot;}
File: /var/www/docker-aio/php/src/Docker/DockerActionManager.php
Line: 588
Trace: #0 /var/www/docker-aio/php/src/Controller/DockerController.php(65): AIO\Docker\DockerActionManager->CreateContainer(Object(AIO\Container\Container))
#1 /var/www/docker-aio/php/src/Controller/DockerController.php(33): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-n...', true)
#2 /var/www/docker-aio/php/src/Controller/DockerController.php(211): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-a...', true)
#3 /var/www/docker-aio/php/src/Controller/DockerController.php(191): AIO\Controller\DockerController->startTopContainer(true)
#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\Controller\DockerController->StartContainer(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#5 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\Handlers\Strategies\RequestResponse->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#7 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#9 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
#10 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(482): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#11 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
#12 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#13 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#14 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#15 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#16 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#17 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#18 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#19 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#20 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#21 /var/www/docker-aio/php/public/index.php(185): Slim\App->run()
#22 {main}
Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.

<img width="512" alt="image" src="https://github.com/nextcloud/all-in-one/assets/167659034/1ef7fc7b-575b-4fb2-afdc-aba02f8e8aa4">

**Comment by szaimen** [comment, marked as answer]
> invalid mount config for type &quot;bind&quot;: bind source path does not exist: /data/mnt

This is your issue

**Reply by sahanasaakshi** [reply]
How to solve this.?

**Reply by sahanasaakshi** [reply]
Thank you it's working

**Comment by sahanasaakshi** [comment]
I have created the directory then run the container is working but when I click on **Open your Nextcloud** , getting below screen.

<img width="439" alt="image" src="https://github.com/nextcloud/all-in-one/assets/167659034/18c17eca-adb2-4c01-bd85-e64c9585d857">


![Screenshot 2024-04-22 105007](https://github.com/nextcloud/all-in-one/assets/167659034/523f528a-2e02-4795-b9b8-4c0177318b47)

**Comment by sahanasaakshi** [comment]
I'm not able to view the Shared folder content in the backend (command line). When I checked it is showing no files or folder. In the frontend can see. Can you please help me on this. How to view a "Shared" folder files/folders in the command line.  And also I have migrated nextcloud from baremetal server to docker AIO. I can only see files but not Shared folder data it is showing empty. Please suggest me.

**Reply by sahanasaakshi** [reply]
Can someone help me on this please.

**Comment by sahanasaakshi** [comment]
Can you please update. Even when I migrate a nextcloud to docker container AIO, that time also didn't get the Shared folder data. Please help on this.

**Comment by sahanasaakshi** [comment]
If yes, can list Shared folder files/folders, using file sharing app in the command line, how to copy those files to another folder..
